meetings/2024-07-10.md

```diff
@@ -0,0 +1,107 @@
+# Node.js Technical Steering Committee (TSC) Meeting 2024-07-10
+
+## Links
+
+* **Recording**:  <https://www.youtube.com/watch?v=5IVNHCsiBaI>
+* **GitHub Issue**: <https://github.com/nodejs/TSC/issues/1592>
+
+## Present
+
+* Benjamin Gruenbaum @benjamingr (voting member)
+* Geoffrey Booth @GeoffreyBooth (voting member)
+* James Snell @jasnell (voting member)
+* Joyee Cheung @joyeecheung (voting member)
+* Chengzhong Wu @legendecas (voting member)
+* Marco Ippolito @marco-ippolito (voting member)
+* Matteo Collina @mcollina (voting member)
+* Michael Dawson @mhdawson (voting member)
+* Rafael Gonzaga @RafaelGSS (voting member)
+* Richard Lau @richardlau (voting member)
+* Robert Nagy @ronag (voting member)
+* Tobias Nießen @tniessen (voting member)
+
+## Agenda
+
+### Announcements
+
+* Rafael - had security releases on Monday, please update
+
+### Reminders
+
+* Remember to nominate people for the [contributor spotlight](https://github.com/nodejs/node/blob/main/doc/contributing/reconizing-contributors.md#bi-monthly-contributor-spotlight)
+
+### CPC and Board Meeting Updates
+
+* No new updates this week
+
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* `npm` is installed wrongly on windows machine [#53538](https://github.com/nodejs/node/issues/53538)
+  * James, talked to Yagiz a bit about this, main reason was to raise visibility.  People are
+    hitting it, problem we is that we don’t have anybody who has volunteered to fix.
+  * Benjamin, Node.js project cannot fix this, needs a fix on the npm side.
+  * Michael, assume there are already issues in the npm repository?
+  * Tobias, yes, we made it a bit harder in a security release related, there is a work around
+    available.
+  * Michael this is only with the windows installer?
+  * Richard, on windows the directory goes somewhere other than where Node.js is installed
+  * Michael, does it make sense to add to documentation?
+  * Richard, doubt it will help much
+  * Tobias, could add to installer, something with a triple check. Do you want to do X, but is a
+    risk. Don’t recommend that.
```

Comment:
```suggestion
  * Tobias, could add option to create the directory to installer, users would have to triple check to avoid security issue. Don't recommend that.
```